GHzDACs/Cleanup/dac.py

```diff
@@ -936,8 +962,252 @@ class DacRunner_Build11(DacRunner_Build7):
 class DAC_Build11(DAC_Build7):
     RUNNER_CLASS = DacRunner_Build11
 
+
 fpga.REGISTRY[('DAC', 11)] = DAC_Build11
 
+
+### Jump table
+
+class DacRunner_Build13(DacRunner_Build7):
+    # noinspection PyMissingConstructor
+    def __init__(self, dev, reps, start_delay, jt_entries, jt_counters, sram):
+        """
+
+        :type dev: DAC_Build13
+        :param reps:
+        :param start_delay:
+        :param jt_entries:
+        :param jt_counters:
+        :param sram:
+        :return:
+        """
+        self.dev = dev
+        self.reps = reps
+        self.start_delay = start_delay
+        self.jump_table = jumpTable.JumpTable(jumps=jt_entries, counters=jt_counters)  # TODO: start address?
+        self.sram = sram
+        self.nPackets = 0  # we don't expect any packets back
+
+    def pageable(self):
+        return False  # no paging for JT
+
+    def loadPacket(self, page, isMaster):
+        """Create pipelined load packet.  For DAC, upload mem and SRAM."""
+        if isMaster:
+            # TODO: how can we add a delay to the JT?
```

Comment:
I don't know, I need to talk to @DanielSank and @JulianSKelly about it. (Good thing I put the todo...?)